boards/arm/apollo4p_evb/apollo4p_evb.dts

```diff
@@ -0,0 +1,14 @@
+/dts-v1/;
+#include <ambiq/ambiq_apollo4p.dtsi>
+
+/ {
+	model = "Ambiq Apollo4 Plus evaluation board";
+	compatible = "ambiq,apollo4p_evb";
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-pipe = &uart0;
```

Comment:
related to my previous commit, where's uart0?

Reply:
After commits reordering `uart0` is now present when the board is added.